From 25246a79f030b97d1a6e5947dc19559187a61248 Mon Sep 17 00:00:00 2001
From: Boyuan Yang <073plan@gmail.com>
Date: Wed, 9 Jan 2019 13:08:54 -0500
Subject: [PATCH] utils/confighandler.cpp: Enable Pin and Text tool by default
 (#443)

This commit enables all the available tools by default so that
users may use them out-of-box without bothering with modifying
configuration.
---
 src/utils/confighandler.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/utils/confighandler.cpp b/src/utils/confighandler.cpp
index a30c412..46a34f7 100644
--- a/src/utils/confighandler.cpp
+++ b/src/utils/confighandler.cpp
@@ -54,7 +54,9 @@ QVector<CaptureButton::ButtonType> ConfigHandler::getButtons() {
                 << CaptureButton::TYPE_SAVE
                 << CaptureButton::TYPE_EXIT
                 << CaptureButton::TYPE_IMAGEUPLOADER
-                << CaptureButton::TYPE_OPEN_APP;
+                << CaptureButton::TYPE_OPEN_APP
+                << CaptureButton::TYPE_PIN
+                << CaptureButton::TYPE_TEXT;
     }
 
     using bt = CaptureButton::ButtonType;
-- 
2.16.4

